setup/ubuntu1804linuxmint18.sh

```diff
@@ -1,18 +1,18 @@
 #!/usr/bin/env bash
 
-# Script to setup an AOSP Build Environment on Ubuntu 16.04 and above and Linux Mint 18.x
+# Script to setup an AOSP Build Environment on Ubuntu 18.04 and above and Linux Mint 18.x
 
 clear
 echo -e "Installing Dependencies!"
 sudo apt update -y
-sudo apt install python gnupg flex bison gperf libsdl1.2-dev libesd0-dev \
+sudo apt install python gnupg flex curl bison gperf libsdl1.2-dev libesd0-dev \
 squashfs-tools build-essential zip libncurses5-dev zlib1g-dev openjdk-8-jre \
-openjdk-8-jdk pngcrush schedtool libxml2 libxml2-utils xsltproc lzop libc6-dev \
-g++-multilib lib32z1-dev lib32ncurses5-dev gcc-multilib liblz4-* pngquant \
+openjdk-8-jdk pngcrush rsync schedtool libssl-dev libwxgtk3.0-dev libxml2 libxml2-utils xsltproc lzop libc6-dev \
+g++-multilib lib32z1-dev lib32ncurses5-dev lib32readline-dev libesd0-dev gcc-multilib liblz4-* pngquant \
```

Comment:
`libesd0-dev` package doesn't exist on Ubuntu 18, AND it's already on the first line of this mammoth package list. Did you actually test this?

Reply:
ya I did test it on my gcloud server  running 18.04